ai_engine_sdk/client.py

```diff
@@ -61,6 +62,11 @@ class FunctionGroup(BaseModel):
     isPrivate: bool
 
 
+class CreateFunctionGroupSchema(BaseModel):
+    name: str
+    isPrivate: bool
```

Comment:
is_private? Just coz python :D

Reply:
Cannot, we should migrate to pydanticv2 and before that adding tests. Nonetheless, task created for all these things.